examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -42,6 +46,16 @@
                                       [-1100, 1100]*u.arcsec,
                                       frame=m.coordinate_frame))
         for m in sunpy.map.Map(files)}
+maps['AIA'].plot_settings['vmin'] = 0  # set the minimum plotted pixel value
+
+
+###############################################################################
+# We will be transforming coordinates where the formation height of 304 A
+# emission makes a difference, so we set the reference solar radius to 4 Mm
+# above the solar surface (see Alissandrakis 2019).
```

Comment:
Do we want a url doi link for this?

Reply:
Probably a good idea.  https://doi.org/10.1007/s11207-019-1552-1 is the DOI URL.  Do we have a style for including such links in gallery examples?

Reply:
I don't think we do but I would need to check. A standard inplace rst url link is good enough for me.

Reply:
Added

Reply:
Thanks.

Reply:
Thanks.